packages/tupaia-web/tsconfig.json

```diff
@@ -15,7 +15,7 @@
     "module": "ESNext",
     "skipLibCheck": true,
     /* Bundler mode */
-    "moduleResolution": "bundler",
+    "moduleResolution": "node",
```

Comment:
Does this break anything with the build?

Reply:
Doesn't seem to but based on our conversation yesterday, I'm going to pull this out for now